contracts/DepositManager.sol

```diff
@@ -36,7 +36,7 @@ contract DepositCore is SubtreeQueue {
     bytes32[] public babyTrees;
     uint256 public depositCount = 0;
 
-    uint256 public MAX_SUBTREE_SIZE;
+    uint256 public govMaxSubtreeSize;
```

Comment:
Before this rename, I often forgot maxSubtreeSize is not a constant and assume it is.

Reply:
Same